mne/io/curry/tests/test_curry.py

```diff
@@ -338,7 +338,8 @@ def sfreq_testing_data(tmpdir, request):
 
     _mock_info_file(src=in_base_name + 'dap', dst=out_base_name + 'dap',
                     sfreq=sfreq, time_step=time_step)
-
+    _mock_info_file(src=in_base_name + 'rs3', dst=out_base_name + 'rs3',
+                    sfreq=sfreq, time_step=time_step)
```

Comment:
This was simply missing, and the mock RS3 file was empty.  This revision simply copies the original RS3 to the testing data folder.